### ISSUE TITLE
I am a win10 system, the outline plug-in can not load successfully

### ISSUE THREAD

**Issue by linjialiang** [facebook-atom/atom-ide-ui#320]
### Description
I am a win10 system, the `outline` plug-in can not load successfully

### Expected Behavior
When you open the `outline panel` using the shortcut key `ctrl-o`, the data should be displayed quickly

### Actual Behavior
I am a win10 system, the `outline` plug-in can not load successfully

![1](https://user-images.githubusercontent.com/8848263/47767061-e3714f00-dd0c-11e8-8474-5ed6b79b4841.png)

### Versions
* Atom:  `1.32.1`
* Client OS: `win10`
* atom-ide-ui: `atom-ide-ui@0.13.0`

### Additional Details
* Installed packages (`apm ls --installed`):

```shell
C:\Users\***>apm ls --installed
Community Packages (49) C:\Users\***\.atom\packages
├── aligner@1.2.4
├── aligner-css@1.2.1
├── aligner-javascript@1.3.0
├── aligner-php@1.2.0
├── atom-beautify@0.33.4
├── atom-clock@0.1.16
├── atom-html-preview@0.2.5
├── atom-ide-ui@0.13.0
├── atom-material-syntax@1.0.8
├── autoclose-html@0.23.0
├── autoprefixer@3.7.1
├── color-picker@2.3.0
├── docblockr@0.13.7
├── double-tag@1.5.0
├── ex-mode@0.18.0
├── file-icons@2.1.26
├── git-plus@8.1.0
├── git-time-machine@2.1.0
├── highlight-selected@0.14.0
├── ide-typescript@0.7.9
├── intentions@1.1.5
├── javascript-snippets@1.2.1
├── jquery-snippets@11.0.0
├── language-apache@1.8.0
├── language-ini@1.20.0
├── language-nginx@0.8.0
├── markdown-preview-enhanced@0.15.8
├── merge-conflicts@1.4.5
├── minimap@4.29.9
├── minimap-bookmarks@0.4.2
├── minimap-cursorline@0.2.0
├── minimap-find-and-replace@4.5.2
├── minimap-git-diff@4.3.1
├── minimap-highlight-selected@4.6.1
├── minimap-selection@4.5.0
├── minimap-split-diff@0.3.7
├── monokai@0.24.0
├── nord-atom-syntax@0.9.1
├── open-in-browser@0.5.2
├── php-ide-serenata@4.5.0
├── platformio-ide-terminal@2.8.4
├── project-manager@3.3.6
├── regex-railroad-diagram@0.19.4
├── relative-numbers@0.9.0
├── split-diff@1.5.3
├── Sublime-Style-Column-Selection@1.7.5
├── sync-settings@0.8.6
├── tree-view-git-status@1.5.2
└── vim-mode-plus@1.35.0
```

**Comment by linjialiang** [comment]
> I'm installing the `php-ide-serenata` plugin, but I've tested it before with the `ide-php` plugin

**Comment by linjialiang** [comment]
> @matthewwithanm @hansonw @ebluestein  @kevinatorchen @aaronabramov

**Comment by WLLR9505** [comment]
Hi @linjialiang, I see this too on ide-typescript, for me isn't a problem, in your print you have a index.php opened in your atom, but you can see the name of file in Italic? this is because if you open other file, its overwrite the active tab in italic, to Outline works correct, double click on active tab to fix the current tab, this works for me all time :slightly_smiling_face:.

![image](https://user-images.githubusercontent.com/22595227/48482292-72936200-e7f7-11e8-91b8-5b85dc0356a4.png)

**Comment by ricpelo** [comment]
> Hi @linjialiang, I see this too on ide-typescript, for me isn't a problem, in your print you have a index.php opened in your atom, but you can see the name of file in Italic? this is because if you open other file, its overwrite the active tab in italic, to Outline works correct, double click on active tab to fix, this works for me all time .

@WLLR9505 , what if you double-click on the file in the TreeView pane? Doing so, the file is opened without italic font in the active tab, but Outline still doesn't work.

**Comment by linjialiang** [comment]
> @WLLR9505 @ricpelo  Thank you, your method is correct, go now can work normally

**Comment by WLLR9505** [comment]
> 
> 
> > Hi @linjialiang, I see this too on ide-typescript, for me isn't a problem, in your print you have a index.php opened in your atom, but you can see the name of file in Italic? this is because if you open other file, its overwrite the active tab in italic, to Outline works correct, double click on active tab to fix, this works for me all time .
> 
> @WLLR9505 , what if you double-click on the file in the TreeView pane? Doing so, the file is opened without italic font in the active tab, but Outline still doesn't work.

@ricpelo So, I think is a problem with your ide-lang package, try to confirm this with DevTools and open a issue on respective package.

**Comment by ricpelo** [comment]
> > > Hi @linjialiang, I see this too on ide-typescript, for me isn't a problem, in your print you have a index.php opened in your atom, but you can see the name of file in Italic? this is because if you open other file, its overwrite the active tab in italic, to Outline works correct, double click on active tab to fix, this works for me all time .
> > 
> > 
> > @WLLR9505 , what if you double-click on the file in the TreeView pane? Doing so, the file is opened without italic font in the active tab, but Outline still doesn't work.
> 
> @ricpelo So, I think is a problem with your ide-lang package, try to confirm this with DevTools and open a issue on respective package.

Several days ago I opened issue #324 in which says the problem can be reproduced using any ide-lang package (I've tested it with `ide-haskell`, `ide-html`, `ide-json`, `ide-yaml` and `ide-php`).

**Comment by WLLR9505** [comment]
> 
> Several days ago I opened issue #324 in which says the problem can be reproduced using any ide-lang package (I've tested it with `ide-haskell`, `ide-html`, `ide-json`, `ide-yaml` and `ide-php`).

@ricpelo Yes I see, so, sorry I can't help you, I use ide-html, ide-css, ide-typescript and ide-json, again, I think is a problem with your package, use the DevTools of your Atom to find what does not work.
